### DISCUSSION TITLE
Command + W Key Binding

### DISCUSSION THREAD

**Discussion by 201flaviosilva** [codesandbox/codesandbox-client#6525] [Q&A] [answered]
Hey there,

This question might seem a bit silly and unnecessary, but is there any wood to force key combinations mostly to `Command+W`, it happens with some frequency.

I want to close a file I'm not using any more, and sometimes I close the whole tab 🥲. As this is a VS Code shortcut, I thought it would make sense to have it here too.

Thanks any help :)

(I already checked the Key Bindings option in the settings and clicked on "Open VS Code Settings", but I didn't find anything related in the VS Code settings :). Sorry if I'm too blind)

**Comment by JamesACS** [comment, marked as answer]
Hey @201flaviosilva - We've heard the request for a CMD + W shortcut before. Unfortunately it's tricky as (as you've encountered) it's also a browser shortcut for closing a tab. Some like having that functionality while using CodeSandbox, others prefer the VS code approach of closing a tab within the interface. 

I'll send this up the chain again on this end, see if we can't add a toggle somewhere👀

**Reply by 201flaviosilva** [reply]
Thank you very much, because it is always useful to have this functionality, maybe optional :)
